docs/schematics/README.md

```diff
@@ -58,17 +58,21 @@ The `@ngrx/schematics` extend the default `@schematics/angular` collection. If y
 Generate the initial state management and register it within the `app.module.ts`
 
 ```sh
-ng generate store State --root --module app.module.ts --collection @ngrx/schematics
+ng generate @ngrx/schematics:store store State --root --module app.module.ts
```

Comment:
This should be `ng generate @ngrx/schematics:store State` without the extra `store`